### ISSUE TITLE
Patchwork armor glitches the entire SSW interface

### ISSUE THREAD

**Issue by PedroAsani** [Solaris-Skunk-Werks/solarisskunkwerks#213]
**Describe the bug**
Patchwork armor glitches when used, rendering the entire interface unusable.

**To Reproduce**
1. Go to 'Armor' tab
2. Click on 'Patchwork Armor'
3. Select any configuration of armor on each location

**Expected behavior**
https://imgur.com/nQIhG70

Moving the mouse around will make the interface look something like the above.

**Screenshots**
Added

**Desktop (please complete the following information):**
 - OS: Win10
 - Version SSW 0.7.5

**Additional context**
Experimental Tech, Clan Invasion, Clan. 50 ton, 3/5/0, Omnimech. Originally testing the Hardened Armor notification.

**Comment by PedroAsani** [comment]
It starts when setting any armor from RT or below.

**Comment by WEKarnesky** [comment]
I am unable to replicate while following the steps listed in the Additional Context section. Can you download a fresh copy of the release candidate and confirm.  There were a temporary versions that missed out on the proper labeling for 7.5,  

https://github.com/Solaris-Skunk-Werks/solarisskunkwerks/releases/tag/v0.7.5-RC1

![image](https://user-images.githubusercontent.com/8506084/100761800-24366900-33a8-11eb-9bb0-7357349c2fab.png)

**Comment by PedroAsani** [comment]
Fresh download provided same result. Interestingly, this is true in windowed mode, but NOT in maximised mode. It might be that the dropdown selection box going outside the java window is causing the issue.

**Comment by PedroAsani** [comment]
Yes, if I shrink the window to minimum I can replicate this with the Cockpit Type selection box as well.

**Comment by PedroAsani** [comment]
Just browsing the issues, and it seems Issue #135 has the same kind of problem. There is a Note about the next pull request fixing it, so I will test again when that has happened (0.7.6?)
